### PR TITLE
Created edit page and added it main route

### PR DESCRIPTION
**Pull request by Shivamyadav07**
edited the note section also

### CODE REVIEW

File: src/Redux/BatchSectionReducer/action.js [hunk 1]
```diff
@@ -42,4 +42,46 @@ export const getSection = () => (dispatch) => {
...
             })
         })
+}
+
+export const getType = () => (dispatch) => {
+    dispatch({
+        type: GET_TYPE_REQUEST,
+        payload: true
+    })
+    return axios.get("")
```
Comment (Shivamyadav07) [inline src/Redux/BatchSectionReducer/action.js:52]: there is some confusion in url once I will clear it update it

File: src/Redux/BatchSectionReducer/action.js [hunk 2]
```diff
@@ -42,4 +42,46 @@ export const getSection = () => (dispatch) => {
...
+            })
+        })
+}
+
+export const getTag = () => (dispatch) => {
+    dispatch({
+        type: GET_TAG_REQUEST,
+        payload: true
+    })
+    return axios.get("")
```
Comment (Shivamyadav07) [inline src/Redux/BatchSectionReducer/action.js:73]: there is some confusion in url once I will clear it update it

File: src/AllRoutes/LectureDetail.jsx [hunk 3]
```diff
@@ -37,14 +44,19 @@ export const LectureDetail = () => {
...
+                            onClick={handleDelete}
+                        >
                             delete
                         </button>
                     </div>
                 </div>
             </header>
             <main>
                 <div className='py-12'>
+                    <DeleteComponent state={isModal} />
```
Comment (mangesh0712) [inline src/AllRoutes/LectureDetail.jsx:59]: change naming to showDeteleModal

File: src/Redux/BatchSectionReducer/action.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ export const getBatch = () => (dispatch) => {
     })
     return axios.get("http://localhost:8888/batch/mybatches")
         .then((r) => {
+            console.log(r);
```
Comment (mangesh0712) [inline src/Redux/BatchSectionReducer/action.js:11]: remove it

File: src/components/CreateLexture/NoteSection.jsx [hunk 5]
```diff
@@ -1,9 +1,61 @@
...
+import React, { useState } from 'react'
+import ReactMarkdown from 'react-markdown'
 
 export const NoteSection = () => {
+    const [writeState, setWriteState] = useState(true);
+    const [previewState, setPreviewState] = useState(false);
+    const [helpState, setHelpState] = useState(false);
+    const [noteValue, setNoteValue] = useState('');
+
+    const handleWrite = () => {
```
Comment (mangesh0712) [inline src/components/CreateLexture/NoteSection.jsx:10]: make a seperate finction for update which is repetative here

**Pull request merged**